src/test/ui/issues/issue-20261.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0282]: type annotations needed
+error[E0282]: type annotations needed in `&(_,)`
```

Comment:
Things seem a bit inconsistent?

Reply:
I think this is the same as https://github.com/rust-lang/rust/pull/61361#discussion_r289202373

Reply:
Yep; continuing over there.